<launch>
  <group ns="slam">
    <node pkg="graph_slam_uk" type="graph_slam2d" name="graph_slam2d" respawn="false" output="screen">
      <!-- <param name="tf_prefix" value="p3dx_0"/> -->
      <param name="robot_base_frame_id" value="base_footprint"/>
      <!-- incomming odometry  tf frame_id without tf_prefix -->
      <param name="odom_farme_id" value="odom_combined"/>
      <!-- calculated map frame_id name -->
      <param name="fixed_farme_id" value="map"/>
      <!-- topic to listen on for odometry messages -->
      <param name="odom_topic" value="/base_odometry/odom"/>
      <!--  where to publish calculated pointcloud -->
      <param name="pcl_map_topic" value="map"/>
      <!-- where to listen for incomming laser msgs -->
      <param name="laser_topic" value="/base_scan"/>
      <!-- Maximal allowe range of laser sensor. Data exceeding this number will be dropped -->
       <param name="maximal_laser_range" type="double" value="30.0"/>
       <param name="optimalizer_iterations" type="int" value="7"/>
       <param name="min_epsilon_change" type="double" value="0.001"/>
       <param name="min_rotated_angle" type="double" value="0.8"/>
       <param name="min_traveled_distance" type="double" value="0.25"/>

       <param name="serialize_graph" type="bool" value="true"/>
       <param name="graph_pub_topic" type="string" value="graph"/>
    </node>
  </group>
</launch>